### ISSUE TITLE
Plant Property And Equipment Value is Incorrect.

### ISSUE THREAD

**Issue by colin99d** [ranaroussi/yfinance#727]
```python
import yfinance as yf
stock = yf.Ticker(ticker)
dfBS = stock.balance_sheet
print(dfBS)
```

returns (among other things)`Property Plant Equipment   4.533600e+10  3.737800e+10  4.130400e+10  3.378300e+10`
The first column is incorrect and does not match Yahoo finance. However, the rest of the years are correct. I have found all the other data returned to be correct.

**Comment by silvavn** [comment]
Not sure which ticker you are referring to, please provide an example.

**Comment by colin99d** [comment]
Sorry, for the example above I used AAPL.

**Comment by ValueRaider** [comment]
Fixed in `yfinance` 0.2
